images/router/clear-route-status.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# This allows for the clearing of route statuses, routers don't clear the routes status so some may be stale.
+# Upon deletion of the routes status active routers will immediatly update with a vaild status
+
+function clear_status() {
+    local namespace="$1"
+    local route_name="$2"
+    MY_JSON_BLOB=$(oc get --raw http://localhost:8001/oapi/v1/namespaces/${namespace}/routes/${route_name}/)
+    MODIFIED_JSON=$(echo "$MY_JSON_BLOB" | jq 'del(.status.ingress)')
+    curl -s -X PUT http://localhost:8001/oapi/v1/namespaces/$1/routes/$2/status --data-binary "${MODIFIED_JSON}" -H "Content-Type: application/json" > /dev/null
```

Comment:
Don't use the positional args, and make sure the `$namespace` here is not shadowing it from outside -- may not even need to pass it through since there will only ever be one in play.